packages/viewport/src/index.ts

```diff
@@ -91,7 +112,9 @@ const mediaQueryLists = {
 	'480px-960px': createMediaQueryList( { min: 480, max: 960 } ),
 };
 
-export function getMediaQueryList( breakpoint ) {
+export function getMediaQueryList(
+	breakpoint: string
+): undefined | false | MinimalMediaQueryList | MediaQueryList {
```

Comment:
with repetitions of this union it could be quite helpful to define it as a thing

```ts
type QueryItem = false | MinimalMediaQueryList | MediaQueryList;

Record<string, QueryItem>

(): QueryItem | undefined => {}
```